silverback/_click_ext.py

```diff
@@ -242,3 +244,16 @@ def get_cluster_client(ctx: click.Context, *args, **kwargs):
         return ctx.invoke(f, *args, **kwargs)
 
     return update_wrapper(get_cluster_client, f)
+
+
+def bot_path_callback(ctx: click.Context, param: click.Parameter, value):
+    path = ctx.params.get("path")
+    if not path:
+        path = "bot:bot"
+    elif ":" not in path:
+        path += ":bot"
+
+    try:
+        return import_from_string(path)
+    except ImportFromStringError:
+        return import_from_string(f"bots.{path}")
```

Comment:
suggestion(non-blocking): I still think you could clean up the error handling here to be more helpful to users. Beyond just the last "attribute not found" error that `import_from_string` raises.